featuretools/primitives/standard/binary_transform.py

```diff
@@ -15,6 +15,18 @@
 
 
 class GreaterThan(TransformPrimitive):
+    """Determines if values X are greater than values Y.
```

Comment:
what are X and Y? this should be "Determines if values in one list are greater than another list". fine to keep x and y in extended description below though

let's apply this rule to all the primitives below too